code/obj/item/go_board.dm

```diff
@@ -0,0 +1,196 @@
+/obj/item/goboard
+	name = "go board"
+	desc = "it's a board for playing go!"
+	icon = 'icons/obj/items/items.dmi'
+	icon_state = "go-board"
+	var/list/openwindows = list()
+	var/list/piecelist = list()
+	var/list/desync = list()
```

Comment:
Is this var used anywhere?

Reply:
I think that was residual from testing, It'll be resolved on the next push :heart: